tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -74,8 +74,10 @@ def wait_for_usb_storage_toggle_ready(self):
         self._wait_for_toggle_ready(*self._usb_storage_checkbox_locator)
 
     def toggle_usb_storage(self):
-        # TODO: remove tap with coordinates after Bug 1061698 is fixed
-        self.marionette.find_element(*self._usb_storage_switch_locator).tap(x=260)
+        # The left hand side of the usb storage switch is overlayed by menuItem-enableStorage
+        # So we do the tapping on the right hand side
+        element = self.marionette.find_element(*self._usb_storage_switch_locator)
+        element.tap(x=(element.size['width']-5))
```

Comment:
We need to keep tracking bug 1061698 here since we use a positional argument to by pass the shadow DOM issue.